src/Telegram.php

```diff
@@ -277,7 +298,7 @@ public function getCommandObject($command)
         $which[] = 'User';
 
         foreach ($which as $auth) {
-            $command_namespace = __NAMESPACE__ . '\\Commands\\' . $auth . 'Commands\\' . $this->ucfirstUnicode($command) . 'Command';
+            $command_namespace = $this->commands_namespace . $auth . 'Commands\\' . $this->ucfirstUnicode($command) . 'Command';
```

Comment:
I'd vote for having this as `$this->commands_namespace . '\\' . $auth . 'Commands\\'`, to not require the final backslashes when setting the namespace.